CHANGELOG/CHANGELOG-1.3.md

```diff
@@ -15,6 +15,7 @@ When cutting a new release, update the `unreleased` heading to the tag being gen
 
 ## unreleased
 
+* [FEATURE] Add token computation function
```

Comment:
We are missing the issue link.

Reply:
There's no public ticket for the issue as far as I know.

Reply:
Indeed, we need to create it. I went ahead and did it: https://github.com/k8ssandra/k8ssandra-operator/issues/728.